src/Components/Voter.jsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react';
+import * as API from '../Api';
+
+export const Voter  = ({votes, target, setVotes}) => {
+    const [vote, setVote] = useState(0);
+    const [thumb, setThumb] = useState('none');
+    const [renderCount, setRenderCount] = useState(-1)
+
+    const handleChange = (input) => {
+
+        if(thumb === 'none'){
```

Comment:
This logic seems a little confusing I think it could probably be refactored later on

Reply:
yes I agree, its the best way I could get it working